app/src/main/scala/org/alephium/app/ServerUtils.scala

```diff
@@ -237,6 +237,34 @@ class ServerUtils(implicit
     }
   }
 
+  def buildSweepMultisig(
+      blockFlow: BlockFlow,
+      query: BuildSweepMultisig
+  ): Try[BuildSweepAddressTransactionsResult] = {
+    val lockupScript = query.fromAddress.lockupScript
+    for {
+      _            <- checkGroup(lockupScript)
```

Comment:
Let's also check the input address is multisig

Reply:
we are actually using [this function](https://github.com/alephium/alephium/blob/a16205506041d2cfd402866b7771d32fafd851b5/app/src/main/scala/org/alephium/app/ServerUtils.scala#L268-L295) right after the `checkGroup` which works only with multsig. 

we will rename `buildUnlockScript `-> `buildMultisigUnlockScript` to be explicit